backends/redis.go

```diff
@@ -0,0 +1,118 @@
+package backends
+
+import (
+	"fmt"
+	"net/url"
+	"strconv"
+
+	"github.com/go-redis/redis"
+)
+
+type RedisBackend struct {
+	client *redis.Client
+}
+
+func NewRedisBackend(connString string) (*RedisBackend, error) {
+	u, _ := url.Parse(connString)
```

Comment:
I like the [Celery-style|http://docs.celeryproject.org/en/latest/getting-started/brokers/redis.html] myself: 

```
redis://:password@hostname:port/db_number
```